extensions/wikia/AdEngine/js/spec/slot/service/srcProvider.spec.js

```diff
@@ -129,7 +129,7 @@ describe('ext.wikia.adEngine.slot.service.srcProvider', function () {
 
 	it('doesn\'t set src=premium if article isn\'t premium', function () {
 		mockContext({'opts.premiumOnly': false});
-		expect(getModule().get('abc')).not.toBe('premium');
+		expect(getModule().get('abc', {})).not.toBe('premium');
```

Comment:
Are these changes required? I don't see any changes related to `srcProvider` now ;)

Reply:
:> They're not necessary right now, reverted.